src/api/v1/db/oracledb/pizzas-dao.js

```diff
@@ -217,4 +248,104 @@ const getPizzaById = async (pizzaId, query) => {
   }
 };
 
-export { getPizzas, getPizzaById };
+/**
+ * Generate queries and bind params for an ingredients relationship resource collection
+ *
+ * @param {object[]} ingredientRelationshipData the `data` key of
+ * `body.data.relationships.ingredients`
+ * @returns {object[]} an object with `queries` and `bindParams` members
```

Comment:
good idea, I'll add some `@throws` documentation

Reply:
Since this function should only return a promise, I'm not sure if we want `@throws` since it won't actually throw any exceptions, just return rejected promises.

Reply:
yep, that checks out. It would be nice if jsdoc had something like an `@resolves` and `@rejects` tag for promises.